### DISCUSSION TITLE
Best Resources to Learn about NPM Packages.

### DISCUSSION THREAD

**Discussion by Sonichigo** [EddieHubCommunity/support#2256] [General]
This are some that i've found interesting: 
- https://www.freecodecamp.org/news/how-to-make-a-beautiful-tiny-npm-package-and-publish-it-2881d4307f78/

- https://docs.npmjs.com/creating-node-js-modules

- https://www.section.io/engineering-education/npm-packages/

**Comment by KukretiShubham** [comment]
Thank you @Sonichigo

**Comment by Prathamesh-Shanbhag** [comment]
Thank you so much @Sonichigo! This is super useful

**Comment by shreemaan-abhishek** [comment]
I have been following [Pankaj](https://twitter.com/heypankaj_) and they regularly share about very cool npm packages.
